README.md

```diff
@@ -0,0 +1,66 @@
+# Sortition Pools
+
+Sortition pool is a logarithmic data structure used to store the pool of
+eligible operators weighted by their stakes. In the Keep network the stake
+consists of staked KEEP tokens. It allows to select a group of operators based
+on the provided pseudo-random seed.
+
+Each privileged application has its own sortition pool and is responsible for
+maintaining operator weights up-to-date.
+
+## In-Depth Reading
+
+To familiarize yourself with the sortition pool and it's design, we provide
+
++ [Building Intuition](docs/building-intuition.md)
++ [Implementation Details](docs/implementation-details.md)
+
+[Building Intuition](docs/building-intuition.md) starts the reader from the
+problem description and an easy-to-understand naive solution, and then works
+its way up to the current design of the sortition pool through a series of
+optimizations.
+
+[Implementation Details](docs/implementation-details.md) builds off of the
+knowledge in [Building Intuition](docs/building-intuition.md), and gets into
+the finer points about the data structure, data (de)serialization, how
+operators join/leave the pool, and how it all comes together to select a full
+group.
+
+## Important Facts
+
++ The max number of operators is `2,097,152`
++ The sortition pool is [optimistic](#optimisic-group-selection)! The on-chain
+  code is only run in the case of a challenge.
+
+FIXME: fill this in with information about:
+
++ how long it takes to create a group of 100 operators
```

Comment:
How long it takes is specific to ECDSA/Beacon, I wouldn't care about it here.

Reply:
[e4bdbea](https://github.com/keep-network/sortition-pools/pull/169/commits/e4bdbeab9f8b298e6b90985b043f9c3d9460dd85)